frameworks/hdfs/universe/resource.json

```diff
@@ -4,6 +4,7 @@
       "jre-tar-gz": "{{jre-jce-unlimited-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.11.0.tar.gz",
+      "hdfs-jre-tar-gz": "{{jre-jce-unlimited-url}}",
```

Comment:
The former is the scheduler's JRE, and the latter is the task's JRE (which will be free from any framework-side constraints once we transition fully to the default executor). It seemed like it would cause confusion down the road if these two were coupled, since they have effectively nothing to do with each other, even though right now there is not a pressing reason to bring in two different JRE instances.